content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -307,6 +307,18 @@ If expanding underlying storage fails, the cluster administrator can manually re
 4. Re-create the PVC with smaller size than PV and set `volumeName` field of the PVC to the name of the PV. This should bind new PVC to existing PV.
 5. Don't forget to restore the reclaim policy of the PV.
 
+##### User recovery expansion failure
```

Comment:
It looks like there are two ways to attempt recovery when expansion fails.

I would use https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs to show the two choices.